Sources/MapboxMaps/Annotations/ClusterOptions.swift

```diff
@@ -0,0 +1,73 @@
+/// Options to show and configure point annotation clustering with using PointAnnotationManager.
+///
+/// It exposes a minimal of configuration options, a more advanced setup can be created manually with
+/// using CircleLayer and SymbolLayers directly.
+
+public struct ClusterOptions {
+
+    /// Define a set of cluster options to determine how to cluster annotations.
+    /// Providing clusterOptions when initializing a ``PointAnnotationManager``
+    /// will turn on clustering for that ``PointAnnotationManager``.
+    public init(clusterRadius: Double = 50,
```

Comment:
nit: Place init after property declarations(at least that seems to be the pattern).